core/cas-server-core-events/src/test/java/org/apereo/cas/support/events/CasAuthenticationEventListenerTests.java

```diff
@@ -55,6 +62,14 @@
 })
 @Tag("Events")
 public class CasAuthenticationEventListenerTests {
+    private static final String REMOTE_ADDR_IP = "123.456.789.010";
+    private static final String LOCAL_ADDR_IP = "123.456.789.000";
+    public static final int NUM_TO_USE_IP1 = 10;
+    public static final int THREAD_POOL_SIZE = 10;
+    public static final int NUM_OF_REQUESTS = 20;
+
+    private final static AtomicInteger numEventsReceived = new AtomicInteger(0);
```

Comment:
This is also not a good idea at all. A static field almost always messes with tests parallel execution. You should redesign your tests so they run independently, OR, you should not rely on a static counter.

Reply:
I refactored a bit, and removed the need for this.